domains/certificates/Query.API/API.AppTests/Testcontainers/RabbitMqContainer.cs

```diff
@@ -0,0 +1,41 @@
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using API.RabbitMq.Configurations;
+using Testcontainers.RabbitMq;
+using Xunit;
+
+namespace API.AppTests.Testcontainers;
+
+public partial class RabbitMqContainer : IAsyncLifetime
+{
+    private readonly global::Testcontainers.RabbitMq.RabbitMqContainer testContainer;
```

Comment:
Why is this long type needed?

Reply:
We have a class in our namespace named RabbitMqContainer, which is using a class named RabbitMqContainer in another namespace.